bert/yamls/glue/mcloud_run.yaml

```diff
@@ -0,0 +1,105 @@
+# This YAML is intended to serve as a reference for running on the Mosaic Cloud
+# The config from `glue/mosaic-bert-base-uncased.yaml` is copied into the `parameters` field below.
+# You can copy/modify the contents of this file to run different workloads, following the other
+# examples in this directory.
+#
+# Note that some of the fields in this template haven't been filled in yet.
+# Please resolve any `null` fields before launching!
+#
+# When ready, use `mcli run -f yamls/glue/mcloud_run.yaml` to launch
+
+run_name: &run_name mosaic-bert-base-uncased-glue-finetuning
+cluster: null  # Name of the cluster to use for this run
+gpu_type: a100_80gb # Type of GPU to use (we use a100_80gb)
+gpu_num: 8  # Number of GPUs to use
+image: mosaicml/composer:0.11.1
+integrations:
+  - integration_type: "git_repo"
+    git_repo: mosaicml/benchmarks
+command: |
+  cd benchmarks/bert
+  pip install -r requirements.txt
+  composer main.py /mnt/config/parameters.yaml
+
+# `parameters` copied from `glue/mosaic-bert-base-uncased.yaml`
+parameters:
```

Comment:
Would still be great to not duplicate all the params in these files. Shouldn't it work without this by just having the command point to the regular yaml?

Reply:
Yes, it would work if we instead did `composer main.py yamls/main/___.yaml` in the command, but that would require  having to commit/push changes to `yamls/main/___.yaml` before running it using `mcli`. I personally find that very tedious, which is why I have it set up this way.

Reply:
ah, so I think the current approach is screwed whether we do this or not. Right now these mcli yamls *only* work if you're both:

1. checking in all your changes
2. pushing your code to `mosaicml/benchmarks` 

What we actually need to do is switch to using an MCLI local integration, which will "just work" no matter where the user's code is / what files they change.

I'm planning on switching everything to local integrations in the next couple days unless some NLP person has bandwidth to take this on (in theory should take like 10min).

Reply:
I'm considering switching to a local integration out of scope for this PR though.